apps/drec-api/.eslintrc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   parserOptions: {
     project: 'tsconfig.json',
+    tsconfigRootDir: __dirname,
```

Comment:
that helps with something?

Reply:
Yep. Eslint was not working properly and this fixed it. Maybe because I am using WSL..not sure. But it fixed it

Reply:
I will try to apply this locally as well